frontend/src/main/pages/Restaurants/RestaurantDetailsPage.js

```diff
@@ -0,0 +1,19 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { useParams } from "react-router-dom";
+import RestaurantTable from 'main/components/Restaurants/RestaurantTable';
+import { restaurantUtils } from 'main/utils/restaurantUtils';
+
+export default function RestaurantDetailsPage() {
+  let { id } = useParams();
+
+  const response = restaurantUtils.getById(id);
+
+  return (
+    <BasicLayout>
+      <div className="pt-2">
+        <h1>Restaurant Details</h1>
+        <RestaurantTable restaurants={[response.restaurant]} showButtons={false} />
```

Comment:
Changing this to `[response.restaurant || {}]` will make it work with the storybook template from ucsbdates

Reply:
Nevermind, it's too hard to add test coverage to this for now. Better to do it when transitioning to backend.